### PR TITLE
[14.0][FIX] [URGENT] generate pain address block that is supported by german banks

### PR DESCRIPTION
**Pull request by luc-demeyer**

### CODE REVIEW

**Review by fkantelberg — APPROVED** [review]

**Comment by rafaelbn** [comment]
Please @OCA/banking-maintainers . Thank you!

File: account_banking_pain_base/models/account_payment_order.py [hunk 1]
```diff
@@ -540,7 +519,34 @@ def generate_address_block(self, parent_node, partner, gen_args):
                     70,
                     gen_args=gen_args,
                 )
-
+            if (
+                gen_args.get("pain_flavor").startswith("pain.001.001.")
+                or gen_args.get("pain_flavor").startswith("pain.008.001.")
```
Comment (pedrobaeza) [inline account_banking_pain_base/models/account_payment_order.py:524]: Which one is `pain.008`?

Comment (luc-demeyer) [reply account_banking_pain_base/models/account_payment_order.py:524]: @pedrobaeza 
I don't know if I understand your question correctly.
pain.008 is for SEPA Direct Debit.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
OK, let's merge it: /ocabot merge patch

**Pull request merged**